src/main/java/hudson/plugins/ansicolor/ColorConsoleAnnotator.java

```diff
@@ -50,12 +50,10 @@
     private static final long serialVersionUID = 1;
 
     private final String colorMapName;
-    private final String charset;
```

Comment:
I guess this could cause issues if someone upgraded the plugin to the version with this fix and then downgraded to 0.6.0, but it's not clear to me what the lifecycle of a serialized `ConsoleAnnotator` would be to know for sure if it would matter.

Reply:
It is only serialized while a `console` page for a running build is being held open, so I do not think you need to care.